hotspot_env.yaml

```diff
@@ -0,0 +1,30 @@
+name: vision_hotspot
+channels:
+  - defaults
+dependencies:
+  - pip=20.1.1
+  - python=3.8.3
+  - pip:
+    - cycler==0.10.0
+    - git+https://github.com/yoseflab/Hotspot.git
+    - joblib==0.15.1
+    - kiwisolver==1.2.0
+    - llvmlite==0.33.0
+    - matplotlib==3.2.2
+    - numba==0.50.1
+    - numpy==1.19.0
+    - pandas==1.0.5
+    - patsy==0.5.1
+    - pyparsing==2.4.7
+    - python-dateutil==2.8.1
+    - pytz==2020.1
+    - scikit-learn==0.23.1
+    - scipy==1.5.0
+    - seaborn==0.10.1
+    - six==1.15.0
+    - statsmodels==0.11.1
+    - threadpoolctl==2.1.0
+    - tqdm==4.46.1
+    - ete3
+prefix: /Users/yanay/Desktop/VISION/envs
```

Comment:
I don't think we want this `hotspot_env.yaml` to be tracked. We should just be clear that users should install Hotspot prior using the instructions on the Hotspot tutorial website.